manifests/vhost.pp

```diff
@@ -1705,6 +1705,7 @@
   $proxy_dest_reverse_match                                                         = undef,
   $proxy_pass                                                                       = undef,
   $proxy_pass_match                                                                 = undef,
+  $proxy_requests                                                                   = false,
```

Comment:
Perhaps this should be of the datatype `Boolean`?

Reply:
Thanks for the remark. I've added 'Optional[Boolean]' as data type for proxy_requests variable.

Reply:
Why `Optional`? There's a default (`false`) so there's no way to make it `undef`.

Reply:
Okay, fixed that. Thanks for your patience

Reply:
No worries :) For my understanding (and to more effectively do reviews in the future), what made you choose `Optional`? I've seen others choose `Optional[Boolean]` because they thought it meant the value could be `false` then. Was it that or something else?

Reply:
Well, the way in which the template handles the boolean, if it's not set; the value will become 'false', so I figured setting the boolean would be optional. It's kinda new to me.

Reply:
Ok, good to know. In general I try to enforce clean data types as input. The template can be more robust than the actual input.

Reply:
I see. So concerning this pull request, is there anything you're expecting from me before this feature could go into the main module?

Reply:
I'm not part of the puppetlabs org, just a community member so I can't actually say :) It does look like you should regenerate the REFERENCE.md file so it has the correct data type.

Reply:
Fixed that